src/components/star-rating.yml

```diff
@@ -0,0 +1,4 @@
+starRating:
+  totalCountText:
+    en: "{totalCount, plural, =0 {no star ratings yet} =1 {# star rating} other {# star ratings}}"
+    de: "{totalCount, plural, =0 {noch keine Sternebewertungen} =1 {# Sternebewertung} other {# Sternebewertungen}}"
```

Comment:
I would suggest dropping the "star" from "star rating", as rating is the feature and star is just the representation. It sounds a bit much to me. Maybe even dropping "yet" / "noch".

Reply:
After seeing it, I'm even wondering if maybe we should just put the count in parenthesis and have a tooltip saying something like "Number of ratings" or "X users have given a rating"? What do you think? 🤔

Reply:
I have reworked the styling based on our discussion.